src/common/MonotonicCounter.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2016 The Android Open Source Project
```

Comment:
2018

Reply:
The code was written in 2016. I copied it from inside AAudio.